<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>index.knit</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="website.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"><img src="Images/Lab_Logo_White.png" alt="drawing" height="55" style="margin-top: -15px;"/></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav navbar-right">
        <li><a href="#HOME">Home</a></li>
        <li><a href="#PEOPLE">People</a></li>
        <li><a href="#RESEARCH">Research</a></li>
        <li><a href="#PUBLICATIONS">Publications</a></li>
        <li><a href="#NEWS">News</a></li>
        <li><a href="#OPPORTUNITIES">Opportunities</a></li>
        <li><a href="#FUNDING">Funding</a></li>
        <li><a href="#CONTACT">Contact</a></li>
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
            Teaching
     
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu" role="menu">
            <li>
              <a href="https://noonanm.github.io/Biol520C/index.html">Biology 520C</a>
            </li>
          </ul>
        </li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->




<div id="header">




</div>


<script src="https://kit.fontawesome.com/e77fcc4fb5.js" crossorigin="anonymous"></script>
<header>
<div class="wrap">
<p><a id="HOME"></a> </br></p>
<center>
<p><img src="Images/Lab_Logo_Trans.png" alt="drawing" width="35%" style="margin-top:5%;"/> </br> </br></p>
<div id="full">
<p>The Quantitative Ecology Lab is focused on protecting vulnerable species. We use data driven approaches and cutting-edge statistics to ensure that the <em>evidence</em> used to support evidence based conservation is both accurate and reliable.</p>
</div>
</br> </br> </br>
</center>
</div>
</header>
<p></br> </br></p>
<p></br></p>
<p><a id="PEOPLE"></a> </br></p>
<center>
<h2>
PEOPLE
</h2>
</center>
<center>
<h3>
QUANTITATIVE ECOLOGY LAB HEAD
</h3>
</center>
<center>
<p><img src="Images/MikeN.png" alt="drawing" width="20%"/></p>
<p>Dr. Michael Noonan | <a href="mailto:michael.noonan@ubc.ca"><i class="fas fa-envelope"></i></a> <a href="https://twitter.com/MikeNoonanUBC"><i class="fab fa-twitter"></i></a> <a href="https://scholar.google.com/citations?user=O5Wu4jMAAAAJ&hl=en"><i class="fas fa-graduation-cap"></i></a> <a href="Files/Noonan_CV.pdf" target="_blank"><i class="fas fa-file-alt"></i></a></p>
<p>Assistant Professor, Department of Biology</p>
</center>
<p></br></p>
<center>
<h3>
GRADUATE STUDENTS
</h3>
</center>
<center>
<p><img src="Images/San.png" alt="drawing" width="20%"/></p>
<p>Sandra Cuadros | <a href="mailto:sandra.scd88@gmail.com"><i class="fas fa-envelope"></i></a> <a href="https://twitter.com/sandracuadros11"><i class="fab fa-twitter"></i></a> <a href="https://www.instagram.com/wayra_killari"><i class="fab fa-instagram"></i></a></p>
<p>Doctoral Candidate, Department of Biology</p>
</center>
<p></br></p>
<p><a id="RESEARCH"></a> </br></p>
<p><img src="Images/Trees.png" alt="drawing" width="100%" /></p>
<center>
<h2>
RESEARCH
</h2>
</center>
<p></br></p>
<p>Species conservation is a multi-faceted challenge that requires effort from people with skillsets ranging from applied ecology, economics, social sciences, data science, and biogeography to name only a few. For the efforts of this community to succeed, the evidence that is being used to inform conservation decisions must be accurate and reliable. UBC Okanagan’s Quantitative Ecology Lab is focused on supporting the statistically efficient integration of data into ecological theory and conservation initiatives. To this end, the lab’s work follows a step-wise process of methods development, testing, refinement, and application structured around two separate, but complimentary lines of research. The first falls under an umbrella called ‘Biology or bias’, and is aimed at understanding when/why different statistical approaches lead to differing conclusions, and how to avoid estimation bias. The second focuses on macro-ecology and global conservation by pairing high quality data with cutting edge analytical tools.</p>
<div id="biology-or-bias" class="section level2">
<h2>Biology or Bias</h2>
<p>Technological advances are transforming ecology as a field. Ecologists are now in a position to collect more accurate data for a larger number of species and individuals, and in more ecosystems than ever before. Problematically, however, statistical techniques for analysing these new and challenging datasets often lag behind. Worse, pairing these data with conventional statistical tools can lead to biased and potentially misleading results. With these issues in mind, the Quantitative Ecology Lab aims to bridge the gap between technological advances and practical ecology. Using carefully tailored experiments, we aim to clarify when analyses will work, when they should be avoided, what conditions cause them to fail, and to what extent that failure might be. This work provides researchers with the novel tools and practical guidelines needed to reliably answer ecological and conservation relevant questions from challenging and nuanced datasets. </br> </br></p>
<p><img src="Images/Banner.png" alt="drawing" /> </br> </br></p>
</div>
<div id="macro-ecology-and-global-conservation" class="section level2">
<h2>Macro-ecology and Global Conservation</h2>
<p>The Quantitative Ecology lab has established an international team, comprised of &gt;80 of the world’s leading movement ecologists. Through this network of specialists, we have built one of the largest animal tracking dataset ever assembled, consisting of 1300 individuals representing 76 mammalian and avian species. With the average, all-in cost of tracking an animal being <a href="https://doi.org/10.1071/WR10211">∼$9,500</a>, this dataset can be valued at &gt;$11 million. Collectively this unprecedented dataset, and broad pool of expert knowledge make our team uniquely positioned to support global conservation efforts. Our lab uses these data to answer questions about the evolutionary history and function of key aspects of animal movement. We then use these higher level findings to contextualise species’ responses to human induced rapid environmental change (HIREC), and provide actionable findings for species conservation. </br> </br></p>
<center>
<img src="Images/Collaboration_Map_Full.png" alt="drawing" width="90%"/>
</center>
<p><a id="PUBLICATIONS"></a> </br></p>
<p><img src="Images/Road.png" alt="drawing"  width="100%" /></p>
<center>
<h2>
PUBLICATIONS
</h2>
</center>
<p></br></p>
<p><a href="#CONTACT">Contact</a> the lab if you would like PDFs of any of the publications listed on this page.</p>
<p></br></p>
<div id="section" class="section level4">
<h4>2021</h4>
<ul>
<li><p>Medici EP, Mezzini S, Fleming CH, Calabrese JM, &amp; <strong>Noonan MJ</strong>. (2021) Movement ecology of vulnerable lowland tapirs across a gradient of human disturbance. bioRxiv DOI: <a href="https://doi.org/10.1101/2021.11.12.468362" class="uri">https://doi.org/10.1101/2021.11.12.468362</a>.</p></li>
<li><p>Milazzotto MP, <strong>Noonan MJ</strong>, &amp; Ferraz MAMM. (2021) RNAseq analysis reveals dynamic metaboloepigenetic profiles of human, mouse and bovine pre-implantation embryos. bioRxiv DOI: <a href="https://doi.org/10.1101/2021.06.25.449773" class="uri">https://doi.org/10.1101/2021.06.25.449773</a>.</p></li>
<li><p>Fleming CH, Deznabi I, Alavi S, Crofoot MC, Hirsch BT, Medici EP, <strong>MJ Noonan</strong>, Kays R, Fagan WF, Sheldon D, &amp; JM Calabrese JM (2021) Population-level inference for home-range areas. bioRxiv DOI: <a href="https://doi.org/10.1101/2021.07.05.451204" class="uri">https://doi.org/10.1101/2021.07.05.451204</a>.</p></li>
<li><p>Fleming CH, Drescher-Lehman J, <strong>Noonan MJ</strong> (with 34 other authors)…, &amp; Calabrese JM. (2021) A comprehensive framework for handling location error in animal tracking data. bioRxiv DOI: <a href="https://doi.org/10.1101/2020.06.12.130195" class="uri">https://doi.org/10.1101/2020.06.12.130195</a>.</p></li>
<li><p>Silva I, Fleming CH <strong>Noonan MJ</strong>, Alston J, Folta C, Fagan WF, &amp; Calabrese JM. (2021) A semi-variance approach to visualising phylogenetic autocorrelation. Methods in Ecology and Evolution, 0(0), 0 – 0 DOI: <a href="https://doi.org/10.32942/osf.io/23wq7" class="uri">https://doi.org/10.32942/osf.io/23wq7</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Fagan WF, &amp; Fleming CH. (2021) A semi-variance approach to visualising phylogenetic autocorrelation. Methods in Ecology and Evolution, 0(0), 0 – 0 DOI: <a href="https://doi.org/10.1111/2041-210X.13763" class="uri">https://doi.org/10.1111/2041-210X.13763</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Ascensão F, Yogui DR, Desbiez ALJ. Roads as ecological traps for giant anteaters. Animal Conservation, 0(0), 1–13. DOI: <a href="https://doi.org/10.1111/acv.12728" class="uri">https://doi.org/10.1111/acv.12728</a>.</p></li>
<li><p><strong>Noonan MJ</strong>…(11 other authors)…, &amp; Calabrese JM. (2021) Estimating encounter location distributions from animal tracking data. Methods in Ecology and Evolution, 12(7), 1158 – 1173. DOI: <a href="https://doi.org/10.1111/2041-210X.13597" class="uri">https://doi.org/10.1111/2041-210X.13597</a>.</p></li>
<li><p>Calabrese JM, Fleming CH, <strong>Noonan MJ</strong>, &amp; Dong X. (2021) Point-and-click AKDE home range estimation with ctmmweb. Wildlife Society Bulletin, 45(1), 162 – 169. DOI: <a href="https://doi.org/10.1002/wsb.1154" class="uri">https://doi.org/10.1002/wsb.1154</a>.</p></li>
</ul>
</div>
<div id="section-1" class="section level4 tabset">
<h4 class="tabset">2020</h4>
<ul>
<li><p>Ferraz MAMM, Nagashima JB, <strong>Noonan MJ</strong>, Crosier AE, &amp; Songsasen N. (2020) Oviductal extracellular vesicles improve red wolf and cheetah sperm function post-thawing. International Journal of Molecular Science, 21(10), 3733; DOI: <a href="https://doi.org/10.3390/ijms21103733" class="uri">https://doi.org/10.3390/ijms21103733</a>.</p></li>
<li><p>Ferraz MAMM, Fujihara M, Nagashima JB, <strong>Noonan MJ</strong>, Inoue-Murayama M &amp; Songsasen N. (2020) Cat follicular extracellular vesicles contain proteins regulating cell signaling pathways that enhance meiotic resumption of vitrified oocytes. Scientific Reports, 10, 8619; DOI: <a href="https://doi.org/10.1038/s41598-020-65497-w" class="uri">https://doi.org/10.1038/s41598-020-65497-w</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Fleming CH, …(76 other authors)…, and Calabrese JM. (2020) Effects of body size on estimation of mammalian area requirements. Conservation Biology, 34(3), 1017 – 1028; DOI: <a href="https://doi.org/10.1111/cobi.13495" class="uri">https://doi.org/10.1111/cobi.13495</a>.</p></li>
</ul>
<div id="section-2" class="section level5">
<h5>2019</h5>
<ul>
<li><p><strong>Noonan MJ</strong>, Fleming CH, Akre T, Dresher-Lehman J, Gurarie E, Harrison AL, Kays R &amp; Calabrese JM. (2019) Scale-insensitive estimation of speed and distance travelled from animal tracking data. Movement Ecology, 7(35), 1 – 15; DOI: <a href="https://doi.org/10.1186/s40462-019-0177-1" class="uri">https://doi.org/10.1186/s40462-019-0177-1</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Tinnesand HV, Müller CT, Rosell F, Macdonald DW, &amp; Buesching CD. (2019) Knowing me, knowing you: anal gland secretion of European badgers (<em>Meles meles</em>) codes for individuality, sex and social group membership. Journal of Chemical Ecology, 45, 823 – 837; DOI: <a href="https://doi.org/10.1007/s10886-019-01113-0" class="uri">https://doi.org/10.1007/s10886-019-01113-0</a>.</p></li>
<li><p>Fleming CH, __Noonan MJ_, Medici E, &amp; Calabrese, JM. Overcoming the challenge of small effective sample sizes in home-range estimation. (2019) Methods in Ecology and Evolution, 10(10), 1679 – 1689; DOI: <a href="https://doi.org/10.1111/2041-210X.13270" class="uri">https://doi.org/10.1111/2041-210X.13270</a>.</p></li>
<li><p>Ishii H, Yamazaki K, <strong>Noonan MJ</strong>, Buesching CD, Newman C, &amp; Kaneko Y. (2019) Testing cellular-phone enhanced GPS tracking technology for urban carnivores. Animal Biotelemetry, 7(19), 1–13; DOI: <a href="https://doi.org/10.1186/s40317-019-0180-8" class="uri">https://doi.org/10.1186/s40317-019-0180-8</a>.</p></li>
<li><p>Ferraz MAMM, Carothers A, Dahal R, <strong>Noonan MJ</strong>, &amp; Songsasen N. (2019) Oviductal extracellular vesicles interact with the spermatozoon’s head and mid-piece and improves its motility and fertilizing ability in the domestic cat. Scientific Reports, 9(9484), 1–12; DOI: <a href="https://doi.org/10.1038/s41598-019-45857-x" class="uri">https://doi.org/10.1038/s41598-019-45857-x</a>.</p></li>
<li><p>Tucker M, … <strong>Noonan MJ</strong> (with 62 other authors)…, &amp; Mueller T. (2019) Large birds travel farther in homogeneous environments. Global Ecology and Biogeography, 28(5), 1–12; DOI: <a href="https://doi.org/10.1111/geb.12875" class="uri">https://doi.org/10.1111/geb.12875</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, …(53 other authors)…, &amp; Calabrese JM. (2019). A comprehensive analysis of autocorrelation and bias in home range estimation. Ecological Monographs, 89(2):e01344; DOI: <a href="https://doi.org/10.1002/ecm.1344" class="uri">https://doi.org/10.1002/ecm.1344</a>.</p></li>
</ul>
</div>
<div id="section-3" class="section level5">
<h5>2018</h5>
<ul>
<li><p><strong>Noonan MJ</strong>, Newman C, Markham A, Buesching CD, Bilham K, &amp; Macdonald DW. (2018) <em>In situ</em> behavioral plasticity as compensation for weather variability: implications for future climate change. Climatic Change, 149(3-4), 457–471; DOI: <a href="https://doi.org/10.1007/s10584-018-2248-5" class="uri">https://doi.org/10.1007/s10584-018-2248-5</a>.</p></li>
<li><p>Winner K, <strong>Noonan MJ</strong>, Fleming CH, Olson KA, Mueller T, Sheldon D, &amp; Calabrese JM. (2018) Statistical inference for home range overlap. Methods in Ecology and Evolution, 9(7), 1679–1691; DOI: <a href="https://doi.org/10.1111/2041-210X.13027" class="uri">https://doi.org/10.1111/2041-210X.13027</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Tinnesand HV, &amp; Buesching CD. (2018) Normalizing Gas-Chromatography–Mass Spectrometry Data: Method Choice can Alter Biological Inference. BioEssays, 40(6), 1–12; DOI: <a href="https://doi.org/10.1002/bies.201700210" class="uri">https://doi.org/10.1002/bies.201700210</a>.</p></li>
<li><p>Bilham K, Newman C, Buesching CD, <strong>Noonan MJ</strong>, Boyd AC, Smith AL, &amp; Macdonald DW. (2018) The effects of weather conditions on oxidative stress, oxidative damage and antioxidant capacity in a wild-living mammal, the European badger (<em>Meles meles</em>). Physiological and Biochemical Zoology, 91(4), 987–1004; DOI: <a href="https://doi.org/10.1086/698609" class="uri">https://doi.org/10.1086/698609</a>.</p></li>
<li><p>Fleming CH, Sheldon D, Fagan WF, Leimgruber P, Mueller T, Nandintsetseg D, <strong>Noonan MJ</strong>, Olson KA, Setyawan E, Sianipar A, &amp; Calabrese JM. (2018) Correcting for missing and irregular data in home-range estimation, Ecological Applications, 28(4), 1003–1010; DOI: <a href="https://doi.org/10.1002/eap.1704" class="uri">https://doi.org/10.1002/eap.1704</a>.</p></li>
</ul>
</div>
<div id="section-4" class="section level5">
<h5>2017</h5>
<ul>
<li>Johnson PJ, <strong>Noonan MJ</strong>, Kitchener A, Harrington LA, Newman C, &amp; Macdonald DW. (2017) Rensching Cats and Dogs: Feeding ecology and fecundity trends explain variation in the allometry of sexual size dimorphism. Royal Society Open Science, 4(6): 170453; DOI: <a href="https://doi.org/10.1098/rsos.170453" class="uri">https://doi.org/10.1098/rsos.170453</a>.</li>
</ul>
</div>
<div id="section-5" class="section level5">
<h5>2016</h5>
<ul>
<li><strong>Noonan MJ</strong>, Johnson PJ, Kitchener A, Harrington LA, Newman C, &amp; Macdonald DW. (2016) Sexual size dimorphism in musteloids: An anomalous allometric pattern is explained by feeding ecology. Ecology and Evolution, 6(23): 8495–8501; DOI: <a href="https://doi.org/10.1002/ece3.2480" class="uri">https://doi.org/10.1002/ece3.2480</a>.</li>
</ul>
</div>
<div id="section-6" class="section level5">
<h5>2015</h5>
<ul>
<li><p>Tinnesand HV, Buesching CD, <strong>Noonan MJ</strong>, Newman C, Zedrosser A, Rosell F, &amp; Macdonald DW. (2015) Will Trespassers Be Prosecuted or Assessed According to Their Merits? A Consilient Interpretation of Territoriality in a Group-Living Carnivore, the European Badger (<em>Meles meles</em>). PLoS One, 10(7), e0132432; DOI: <a href="https://doi.org/10.1371/journal.pone.0132432" class="uri">https://doi.org/10.1371/journal.pone.0132432</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Newman C, Buesching CD, &amp; Macdonald DW. (2015) Evolution and function of fossoriality in the Carnivora: implications for group-living. Frontiers in Ecology and Evolution, 3(116), 1–14; DOI: <a href="https://doi.org/10.3389/fevo.2015.00116" class="uri">https://doi.org/10.3389/fevo.2015.00116</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Abidur Rahman M, Newman C, Buesching CD, &amp; Macdonald DW. (2015) Avoiding verisimilitude when modelling ecological responses to climate change: The influence of weather conditions on trapping efficiency in European badgers (<em>Meles meles</em>). Global Change Biology, 21(20), 3575–3585; DOI: <a href="https://doi.org/10.1111/gcb.12942" class="uri">https://doi.org/10.1111/gcb.12942</a>.</p></li>
<li><p><strong>Noonan MJ</strong>, Markham A, Newman C, Trigoni N, Buesching CD, Ellwood SA, &amp; Macdonald DW. (2015) A new Magneto-Inductive tracking technique to uncover subterranean activity: What do animals do underground? Methods in Ecology and Evolution, 6(5), 510–520; DOI: <a href="https://doi.org/10.1111/2041-210X.12348" class="uri">https://doi.org/10.1111/2041-210X.12348</a>.</p></li>
</ul>
</div>
<div id="section-7" class="section level5">
<h5>2014</h5>
<ul>
<li><strong>Noonan MJ</strong>, Markham A, Newman C, Trigoni N, Buesching CD, Ellwood SA, &amp; Macdonald DW. (2014) Climate and the Individual: Inter-Annual Variation in the Autumnal Activity of the European Badger (<em>Meles meles</em>). PloS One, 9(1), e83156; DOI: <a href="https://doi.org/10.1371/journal.pone.0083156" class="uri">https://doi.org/10.1371/journal.pone.0083156</a>.</li>
</ul>
</div>
<div id="section-8" class="section level5">
<h5>2012</h5>
<ul>
<li><strong>Noonan MJ</strong>, Grant JWA, &amp; Jackson CD. (2012) A quantitative assessment of fish passage efficiency. Fish and Fisheries, 13(4), 450–464; DOI: <a href="https://doi.org/10.1111/j.1467-2979.2011.00445.x" class="uri">https://doi.org/10.1111/j.1467-2979.2011.00445.x</a>.</li>
</ul>
</div>
</div>
</div>
<div id="section-9" class="section level2 unnumbered">
<h2 class="unnumbered"></h2>
<a id="NEWS"></a> </br>
<center>
<img src="Images/heatmap.png" alt="drawing" width="100%" />
</center>
<center>
<h2>
NEWS
</h2>
</center>
<p></br></p>
<div id="section-10" class="section level4">
<h4>2021</h4>
<ul>
<li><p>Graduate student Sandra Cuadros has been awarded a Graduate Dean’s Entrance Scholarship. Congratulations Sandra!</p></li>
<li><p>Two new members have been officially admitted to graduate programs at UBC-O. Welcome Sandra Cuadros and Ryan Gill!</p></li>
<li><p>Graduate student Stefano Mezzini has been awarded a Graduate Dean’s Entrance Scholarship. Congratulations Stefano!</p></li>
<li><p>Stefano Mezzini has been officially admitted to the MSc program at UBC-O. Welcome Stefano!</p></li>
<li><p>The quantitative Ecology Lab has been awarded an <a href="https://www.nserc-crsng.gc.ca/NSERC-CRSNG/FundingDecisions-DecisionsFinancement/ResearchGrants-SubventionsDeRecherche/ResultsGSCDetail-ResultatsCSSDetails_eng.asp?Year=2021&amp;GSC=1503">NSERC Discovery Grant</a> to support the lab’s work over the next 5 years.</p></li>
</ul>
<p><a id="OPPORTUNITIES"></a> </br></p>
<p><img src="Images/okanagan_campus.jpg" alt="drawing" width="100%" /></p>
<center>
<h2>
OPPORTUNITIES
</h2>
</center>
<p>There are currently no open positions for which we are actively recruiting. However, if you’re interested in joining the Quantitative Ecology Lab, you are welcome to email us your CV, unofficial transcripts, a brief blurb about yourself and your interests, and what aspects of the lab’s research interest you.</p>
<p></br></p>
<p><strong>General guidelines for prospective students</strong>: We expect prospective students to have consulted the general information provided at the Biology Graduate program webpage prior to contacting me. Students in the Quantitative Ecology Lab gain valuable experience in quantitative methods, data management / analysis, <code>R</code> usage, and <code>R</code> package development. These skills and experiences are aimed at helping our students find full time employment and / or further their academic careers.</p>
<p></br></p>
<p>Prospective students are encouraged to seek their own sources of external funding (e.g. NSERC), and to be competitive for internal grants available at UBC.</p>
<p></br></p>
<p>General information about graduate studies at the Okanagan campus and in biology:</p>
<p><a href="https://biology.ok.ubc.ca/graduate/biology/">UBC Okanagan Biology graduate program</a></p>
<p><a href="https://gradstudies.ok.ubc.ca">Graduate School at UBC Okanagan</a> </br>  </p>
<p><a id="FUNDING"></a> </br></p>
<center>
<h2>
FUNDING &amp; PARTNERS
</h2>
</center>
<p></br>  </p>
<div class="full-width">
<center>
<img src="Images/Funding_Partners.png" alt="drawing" width="30%" style="margin-top:3%;"/>
</center>
</div>
</div>
</div>

<footer class="footer">
  
  <a id="CONTACT"></a> </br>
  
  <center> <h2>CONTACT</h2> </center> </br>

<div class="row">
  <div class="column">
      <div style= "float:left;position: relative; right: 20px;">
        <a href="https://goo.gl/maps/i5RkFCywp5KzESeL6" style="color: #fff"><img src="Images/map.png" alt="drawing" height="45"/> </a> 
      </div> Syilx Okanagan Nation Territory </br>
         Department of Biology, University of British Columbia, </br>
        1177 Research Road, Kelowna, BC, V1V 1V7, Canada
  </div>
  
  <div class="column">
      <div style= "float:left;position: relative; right: 20px;">
        <a href="tel:+1-250-258-1118" style="color: #fff"><img src="Images/phone.png" alt="drawing" height="45"/> </a>
      </div>
      <p>250-258-1118</p>
  </div>
  
  <div class="column">
    <div style= "float:left;position: relative; right: 20px;">
      <a href="mailto:michael.noonan@ubc.ca" style="color: #fff"><img src="Images/contact.png" alt="drawing" height="45"/> </a>
    </div>
    <p>michael.noonan@ubc.ca</p>
  </div>
</div>

</footer>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
